timeout.go

```diff
@@ -0,0 +1,42 @@
+package codacytool
+
+import (
+	"context"
+	"errors"
+	"os"
+	"strconv"
+	"time"
+)
+
+func timeoutSeconds() time.Duration {
+	value, exists := os.LookupEnv("TIMEOUT_SECONDS")
```

Comment:
this needs to be done in the entrypoint of the tool (same place where we check flags e.g.)

Reply:
Moved it to `init.go`